Contest_API.md

```diff
@@ -639,26 +642,26 @@ identifiers. It is recommended to choose new identifiers with a suffix
 appended to an existing one. For example `cpp17` to specify the ISO 2017
 version of C++.
 
-| ID         | Name        |
-| ---------- | ----------- |
-| ada        | Ada         |
-| c          | C           |
-| cpp        | C++         |
-| csharp     | C\#         |
-| go         | Go          |
-| haskell    | Haskell     |
-| java       | Java        |
-| javascript | JavaScript  |
-| kotlin     | Kotlin      |
-| objectivec | Objective-C |
-| pascal     | Pascal      |
-| php        | PHP         |
-| prolog     | Prolog      |
-| python2    | Python 2    |
-| python3    | Python 3    |
-| ruby       | Ruby        |
-| rust       | Rust        |
-| scala      | Scala       |
+| ID         | Name        | Extensions           |
+| ---------- | ----------- | -------------------- |
+| ada        | Ada         | adb, ads             |
+| c          | C           | c                    |
+| cpp        | C++         | cc, cpp, cxx, c++, C |
+| csharp     | C\#         | cs                   |
+| go         | Go          | go                   |
+| haskell    | Haskell     | hs                   |
+| java       | Java        | java                 |
+| javascript | JavaScript  | js                   |
+| kotlin     | Kotlin      | kt                   |
+| objectivec | Objective-C | m                    |
+| pascal     | Pascal      | pas                  |
+| php        | PHP         | php                  |
+| prolog     | Prolog      | pl                   |
+| python2    | Python 2    | py                   |
+| python3    | Python 3    | py                   |
+| ruby       | Ruby        | rb                   |
+| rust       | Rust        | rs                   |
+| scala      | Scala       | scala                |
```

Comment:
The description of the table needs to be updated. It should mention "extensions" if it includes them.

I think we should also add the entry point name to this table.

The existence of this table implies that there are sensible defaults for all (current) elements. This means we *could* make them all optional.

Reply:
Added the entry point names from [this table](https://icpc.io/problem-package-format/spec/problem_package_format#programs). Also updated the description.

I'm not a fan of making all elements optional. I don't see a big advantage to it and it does mean all clients need this same list somewhere to make it work.